docs/gettingstarted/versioning.md

```diff
@@ -1,7 +1,10 @@
 Versioning
 ==========
 
-In general projects, [Semantic Versioning][semver] is often used (which has the format `MAJOR.MINOR.PATCH`). However, in the case of modding it may be more beneficial to use the format `MCVERSION-MAJORMOD.MAJORAPI.MINOR.PATCH` to be able to differentiate between world-breaking and API-breaking changes of a mod.
+In general projects, [semantic versioning][semver] is often used (which has the format `MAJOR.MINOR.PATCH`). However, in the case of modding it may be more beneficial to use the format `MCVERSION-MAJORMOD.MAJORAPI.MINOR.PATCH` to be able to differentiate between world-breaking and API-breaking changes of a mod.
+
+!!! important
+    Forge uses `ComparableVersion` to compare version strings, which is not fully compatible with the Semantic Versioning 2.0.0 spec, such as the 'prerelease' tag.
```

Comment:
```suggestion
    Forge uses Maven Version ranges to compare version strings, which is not fully compatible with the Semantic Versioning 2.0.0 spec, such as the 'prerelease' tag.
```

Reply:
Why are we being less specific here? Forge uses `ComparableVersion` to compare version strings.

Reply:
We should specify the specification, not implementation